src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -170,7 +163,8 @@
   [rules :- Rules
    request :- ring/Request
    name :- schema/Str]
-  (if-let [ { matched-rule :rule matches :matches } (some #(match? % request) rules)]
+  (if-let [{matched-rule :rule matches :matches}
+           (some #(match? % request) (sort-rules rules))]
```

Comment:
This might be a bit of a pre-optimization, but I'm wondering if it would be better to do the rule sort one time - behind `transform-code`, before they are stored into the Trapperkeeper service config - rather than here.  I think doing it here would mean that the rules would end up being resorted every time an authorization check is done?